### PR TITLE
Addition of httparty simple reading of data

### PR DESCRIPTION
**Pull request by jchapin**

### CODE REVIEW

**Comment by jchapin** [comment]
@code-for-nashville/openaccountability ready for review.

screen shot 2015-06-06 at 11 31 10 am

File: .gitignore [hunk 1]
```diff
@@ -12,10 +12,6 @@ capybara-*.html
 rerun.txt
 pickle-email-*.html
 
-# TODO Comment out these rules if you are OK with secrets being uploaded to the repo
-config/initializers/secret_token.rb
-config/secrets.yml
-
```
Comment (jchapin) [inline .gitignore:18]: Removing this from the Github .gitignore... just for the quick and dirty... get everyone productive approach.

File: Gemfile [hunk 2]
```diff
@@ -22,6 +22,7 @@ gem 'turbolinks'
 gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
+gem 'httparty'
```
Comment (jchapin) [inline Gemfile:25]: This is the library we're going to use to snag API information. It's quick and dirty.

**Comment by ebrengle** [comment]
Nice thumbs up Andy. #JustGoingForIt

**Pull request merged**